schema/v1.1.0/api_models.yaml

```diff
@@ -59,6 +59,67 @@ classes:
   # ============================================================================
   # Entities
   # ============================================================================
+  Alignment:
+    mixins:
+      - IDMixin
+    description: Tiltseries Alignment
+    attributes:
+      annotation_files:
+        range: AnnotationFile
+        inverse: AnnotationFile.alignment
+        multivalued: true
+      per_section_alignments:
+        range: PerSectionAlignment
+        inverse: PerSectionAlignment.alignment
+        multivalued: true
+      deposition:
+        range: Deposition
+        inverse: Deposition.alignments
+      tiltseries:
+        range: Tiltseries
+        inverse: Tiltseries.alignments
+      tomograms:
+        range: Tomogram
+        inverse: Tomogram.alignment
+        multivalued: true
+      run:
+        range: Run
+        inverse: Run.alignments
+      alignment:
+        exact_mappings:
+          - cdp-common:api_alignment
+      alignment_type:
+        exact_mappings:
+          - cdp-common:alignment_alignment_type
+      volume_x_dimension:
+        exact_mappings:
+          - cdp-common:alignment_volume_x_dimension
+      volume_y_dimension:
+        exact_mappings:
+          - cdp-common:alignment_volume_y_dimension
+      volume_z_dimension:
+        exact_mappings:
+          - cdp-common:alignment_volume_z_dimension
+      volume_x_offset:
+        exact_mappings:
+          - cdp-common:alignment_volume_x_offset
+      volume_y_offset:
+        exact_mappings:
+          - cdp-common:alignment_volume_y_offset
+      volume_z_offset:
+        exact_mappings:
+          - cdp-common:alignment_volume_z_offset
+      volume_x_rotation:
+        exact_mappings:
+          - cdp-common:alignment_volume_x_rotation
+      tilt_offset:
+        exact_mappings:
+          - cdp-common:alignment_tilt_offset
+      local_alignment_file:
```

Comment:
I believe this is different from the actual alignment. If that is a correct assumption, should we also add fields to capture the file path to the alignment file?